platform/frontend/src/lib/role.query.ts

```diff
@@ -123,7 +123,9 @@ export function useDeleteRole() {
  * Filters out predefined roles and returns only custom ones
  */
 export function useCustomRoles() {
-  const userIsAuthenticated = useIsAuthenticated();
```

Comment:
_nit_ I see we only use [`useIsAuthenticated`](https://github.com/archestra-ai/archestra/blob/jorlando/pin-better-auth-version/platform/frontend/src/lib/auth.hook.ts#L3) now in `app/_parts/sidebar.tsx` -- is there a difference between using that hook vs calling `authClient.useSession` here directly?

(wondering if we just remove `useIsAuthenticated`, open-ended question)